### ISSUE TITLE
Search for "generated" columns is to greedy

### ISSUE THREAD

**Issue by eeggenberger** [nixys/nxs-data-anonymizer#33]
I had a problem with a Mysql dump where one column wasn't processed properly. It looked like this in the CREATE TABLE statement:

```
  `provider` varchar(50) CHARACTER SET ascii NOT NULL DEFAULT 'system' COMMENT 'The name of the module that generated this link.',
```

Looks like dhCreateTableColumnAdd searches for the GENERATED keyword, but doesn't care if it is an option for the column or just part of the comment.

**Comment by borisershov** [comment]
Hi! Your issue will be fixed in next release

**Comment by borisershov** [comment]
Hi! Fixed in [v1.10.1](https://github.com/nixys/nxs-data-anonymizer/releases/tag/v1.10.1)
